libkbfs/journal_server.go

```diff
@@ -93,6 +98,22 @@ func makeJournalServer(
 	return &jServer
 }
 
+func (j *JournalServer) getDirLocked() string {
+	// TODO: This may end up making paths too long for some
+	// systems. We may end up having to drop the UID (since a
```

Comment:
Went with dropping the UID for now. If tests still pass, I'll do the device ID conversion (as well as maybe trimming the block and md ID path components) in a future PR.